app/controllers/api/projects.php

```diff
@@ -897,14 +899,15 @@
     ->label('sdk.response.model', Response::MODEL_WEBHOOK)
     ->param('projectId', '', new UID(), 'Project unique ID.')
     ->param('name', null, new Text(128), 'Webhook name. Max length: 128 chars.')
+    ->param('enabled', true, new Boolean(), 'Enable or disable a webhook.', true)
```

Comment:
Can we have test for this? To mark webhook disabled, and trigger it's event. Then ensure the webhook didn't run (we have request catcher that can help for this)

Reply:
lets also make sure there is strict validation for the Boolean Validator. Boolean(true)